test/check-metadata.bash

```diff
@@ -0,0 +1,7 @@
+exec > /dev/null 2>&1
+
+if [ "$BUILD_METADATA" = "the first metadata;the second metadata;" ]; then
```

Comment:
Is it worth trimming whitespace _and_ trailing semicola?

Reply:
I don't think so.  I had actually considered _adding_ a semicolon between files, but ultimately decided that I only want to manipulate the data as little as possible.  Trimming newlines seemed necessary because I'm concatenating the files and didn't want trailing newlines to hamper the consumer at the other end.